src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -192,381 +191,387 @@ the MIT License. See LICENSE in the project root for license information. -->
                                           Visibility="{Binding ElementName=UseAcrylicCheckBox, Path=IsChecked, Mode=OneWay}">
-                            <StackPanel x:Name="AcrylicOpacityControl">
-                                <TextBlock x:Uid="Profile_AcrylicOpacity"
+                                <StackPanel x:Name="AcrylicOpacityControl">
+                                    <TextBlock x:Uid="Profile_AcrylicOpacity"
                                            Style="{StaticResource SliderHeaderStyle}"/>
-                                <Grid Style="{StaticResource CustomSliderControlGridStyle}">
-                                    <Grid.ColumnDefinitions>
-                                        <ColumnDefinition Width="*"/>
-                                        <ColumnDefinition Width="Auto"/>
-                                    </Grid.ColumnDefinitions>
-                                    <Slider x:Name="AcrylicOpacitySlider"
+                                    <Grid Style="{StaticResource CustomSliderControlGridStyle}">
+                                        <Grid.ColumnDefinitions>
+                                            <ColumnDefinition Width="*"/>
+                                            <ColumnDefinition Width="Auto"/>
+                                        </Grid.ColumnDefinitions>
+                                        <Slider x:Name="AcrylicOpacitySlider"
                                             Grid.Column="0"
                                             Value="{x:Bind State.Profile.AcrylicOpacity, Converter={StaticResource PercentageConverter}, Mode=TwoWay}"/>
-                                    <TextBlock Grid.Column="1"
+                                        <TextBlock Grid.Column="1"
                                                Text="{Binding ElementName=AcrylicOpacitySlider, Path=Value, Mode=OneWay}"
                                                Style="{StaticResource SliderValueLabelStyle}"/>
-                                </Grid>
-                            </StackPanel>
-                        </ContentPresenter>
+                                    </Grid>
+                                </StackPanel>
+                            </ContentPresenter>
+                        </StackPanel>
 
                         <!--Grouping: Window-->
-                        <TextBlock x:Uid="Profile_WindowHeader" Style="{StaticResource GroupingHeader}"/>
+                        <StackPanel Style="{StaticResource PivotStackStyle}">
+                            <TextBlock x:Uid="Profile_WindowHeader" Style="{StaticResource SubtitleTextBlockStyle}"/>
 
-                        <!--Padding-->
-                        <ContentPresenter Style="{StaticResource SettingContainerStyle}">
-                            <TextBox x:Uid="Profile_Padding"
+                            <!--Padding-->
+                            <ContentPresenter>

```

Comment:
Again, if you're using a `ContentPresenter` without styling, I suggest styling it, then overriding the margin plz.